### ISSUE TITLE
User story - Job Post Salary

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#34]
### If applicable, add a related Epic.

Epic: #28 

### User Story

As a **user**, I can **see a job position's salary range** so that I can **decide whether to apply or not**.

### Acceptance Criteria

- Job Post creation should allow for salary input ( #29 )
- Salary inputs should be optional
- If salary is added, it is reflected on the post
- If salary is not added, the omission is reflected on the post

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Ensure salary field is on job post creation form
- [x] Ensure currency field is on form

**Comment by stephendawsondev** [comment]
This was completed with PR #85
